mmv1/products/vmwareengine/PrivateCloud.yaml

```diff
@@ -297,3 +297,14 @@ properties:
     values:
       - :STANDARD
       - :TIME_LIMITED
+      - :STRETCHED
+
+  - !ruby/object:Api::Type::String
+    name: 'preferredZone'
+    description: |
+      Creation timestamp in RFC3339 text format.
```

Comment:
Looks like this descriptions need to be updated

Reply:
My bad, fixed.